lib/api_me/sorting.rb

```diff
@@ -15,35 +15,34 @@ def results
     end
 
     def sort_meta
-      return Hash.new unless sorting?
+      return {} unless sorting?
       {
-        criteria: sort_criteria.is_blank? || sort_criteria === "" ? default_sort_criteria : sort_criteria,
+        criteria: sort_criteria.is_blank? || sort_criteria === '' ? default_sort_criteria : sort_criteria,
         reverse: sort_reverse,
         record_count: scope.size,
-        total_records: scope.total_count,
+        total_records: scope.total_count
       }
     end
 
     protected
 
     def sort(criteria = default_sort_criteria)
-      if sort_reverse === "true"
-        self.scope = scope.order(criteria => :desc)
-      else
-        self.scope = scope.order(criteria => :asc)
-      end
+      self.scope = if sort_reverse === 'true'
+                     scope.order(criteria => :desc)
+                   else
+                     scope.order(criteria => :asc)
+                   end
```

Comment:
I prefer ternary here:
`self.scope = sort_reverse === 'true' ? scope.order(criteria => :desc) : scope.order(criteria => :asc)`
but up to you.

Reply:
This avoids the line length issue.